.github/workflows/docs-pr.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       provide-link-targets: |
         ansible_collections.ansible.builtin.dict2items_filter
         ansible_collections.ansible.builtin.path_join_filter
+        ansible_collections.ansible.utils.update_fact_module
```

Comment:
To avoid documentation build errors. Only references in this list are allowed to be used in module docs. As long as
```
seealso:
  - module: ansible.utils.update_fact
    description: Do something similar in another way.
```
is in the plugins/filter/structure_merging.py docs, this modification is needed.